testsuite/testsuite-jsp/pom.xml

```diff
@@ -49,6 +64,12 @@
             <version>${project.version}</version>
             <executions>
               <execution>
+                <id>package</id>
```

Comment:
Doesn't this configuration come from the parent?

Likewise with `useUberJar`, also should only be activated with profile `uberjar`.